src/main/java/uk/gov/justice/hmpps/prison/values/Money.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.hmpps.prison.values;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class Money {
+
+    private static final Long PENCE_IN_POUND = 100L;
+
+    public enum Currency {
+        GBP("British Pound", "£");
+        public final String name;
+        public final String symbol;
+        Currency(final String name, final String symbol) {
+            this.name = name;
+            this.symbol = symbol;
+        }
+    }
+
+    private final Currency currency = Currency.GBP;
```

Comment:
I've just noticed everywhere else we are pulling the currency from config but we're are hard coding it here.

Reply:
there are no requirements for this, it's there for completeness and hence defaulted to GBP, can be easily changed in the future when there are hard requirements for this

Reply:
I agree there is no requirement but for consistency it would be worth doing.